bindings/python/iota_sdk/client/_utils.py

```diff
@@ -1,20 +1,43 @@
 # Copyright 2023 IOTA Stiftung
 # SPDX-License-Identifier: Apache-2.0
 
+from typing import Optional
+from abc import ABCMeta, abstractmethod
+
 from iota_sdk.types.common import HexStr
 from iota_sdk.types.output import Output
-from typing import Optional
 
 
-class ClientUtils():
+class ClientUtils(metaclass=ABCMeta):
     """Client utility functions.
     """
 
-    def hex_to_bech32(self, hex: HexStr, bech32_hrp: str) -> str:
+    @abstractmethod
+    def _call_method(self, name, data=None):
+        """
+        Sends a message to the Rust library and returns the response.
+        It is abstract here as its implementation is located in `client.py`, which is a composite class.
+
+        Arguments:
+
+        * `name`: The `name` parameter is a string that represents the name of the method to be called.
+        It is used to identify the specific method to be executed in the Rust library.
+        * `data`: The `data` parameter is an optional parameter that represents additional data to be
+        sent along with the method call. It is a dictionary that contains key-value pairs of data. If
+        the `data` parameter is provided, it will be included in the `message` dictionary as the 'data'
+        key.
+
+        Returns:
+
+        The method returns either the payload from the JSON response or the entire response if there is
+        no payload.
+        """
+
+    def hex_to_bech32(self, hex_str: HexStr, bech32_hrp: str) -> str:
```

Comment:
small suggestion: A bit longer but clearer and more consistent with the method right below: `hex_address_to_bech32_address`. Otherwise it's not really clear what `hex_str` is expected to hold.

Also ... maybe put `Args:` section in the docs?